Test/spec/simd/simd_load.wast

```diff
@@ -0,0 +1,180 @@
+    (v128.store            (i32.const 0) (local.get $0))
+    (v128.store  offset=16 (i32.const 1) (local.get $1))
+    (v128.store  offset=32 (i32.const 2) (local.get $2))
+    (v128.bitselect (v128.load (i32.const 0)) (v128.load offset=16 (i32.const 1)) (v128.load offset=32 (i32.const 2)))
+  )
+  (func (export "as-i8x16.all_true-operand") (param $0 v128) (result i32)
+    (local v128)
+    (v128.store (i32.const 0) (local.get $0))
+    (i8x16.all_true (v128.load (i32.const 0)))
+  )
+  (func (export "as-add/sub/sul-operand") (param $a v128) (param $b v128) (result v128)
+    (v128.store offset=0  (i32.const 0) (local.get $a))
+    (v128.store offset=15 (i32.const 1) (local.get $b))
+    (v128.store (i32.const 0) (i8x16.add (v128.load (i32.const 0)) (v128.load offset=15 (i32.const 1))))      ;; 2 2 2 2 + 3 3 3 3 = 5 5 5 5
+    (v128.store (i32.const 0) (i8x16.sub (v128.load (i32.const 0)) (v128.load offset=15 (i32.const 1))))      ;; 5 5 5 5 + 3 3 3 3 = 2 2 2 2
+    (v128.store (i32.const 0) (i8x16.mul (v128.load (i32.const 0)) (v128.load offset=15 (i32.const 1))))      ;; 2 2 2 2 * 3 3 3 3 = 6 6 6 6
+    (v128.load (i32.const 0))                                                                                 ;; 6 6 6 6
+  )
+  (func (export "as-i8x16.shl-operand") (param $0 v128) (param $1 i32) (result v128)
+    (local v128 i32)
+    (v128.store (i32.const 0) (local.get $0))
+    (i8x16.shl (v128.load (i32.const 0)) (local.get $1))
+  )
+  (func (export "as-f32x4.mul-operand") (param $0 v128) (param $1 v128) (result v128)
+    (local v128 v128)
+    (v128.store (i32.const 0)           (local.get $0))
+    (v128.store offset=16 (i32.const 1) (local.get $1))
+    (f32x4.mul (v128.load (i32.const 0))  (v128.load offset=16 (i32.const 1)))
+  )
+  (func (export "as-f32x4.abs-operand") (param $0 v128) (result v128)
+    (local v128)
+    (v128.store (i32.const 0)           (local.get $0))
+    (f32x4.abs (v128.load (i32.const 0)))
+  )
+  (func (export "as-f32x4.min-operand") (param $0 v128) (param $1 v128) (result v128)
+    (local v128 v128)
+    (v128.store (i32.const 0)          (local.get $0))
+    (v128.store offset=16 (i32.const 1)(local.get $1))
+    (f32x4.min (v128.load (i32.const 0)) (v128.load offset=16 (i32.const 1)))
+  )
+  (func (export "as-i32x4.trunc_sat_f32x4_s-operand") (param $0 v128) (result v128)
+    (local v128)
+    (v128.store (i32.const 0) (local.get $0))
+    (i32x4.trunc_sat_f32x4_s (v128.load (i32.const 0)))
+  )
+  (func (export "as-f32x4.convert_i32x4_u-operand") (param $0 v128) (result v128)
+    (local v128)
+    (v128.store (i32.const 0) (local.get $0))
+    (f32x4.convert_i32x4_u (v128.load (i32.const 0)))
+  )
+  (func (export "as-v8x16.shuffle1-operand") (param $0 v128) (param $1 v128) (result v128)
+    (local v128 v128)
+    (v128.store (i32.const 0) (local.get $0))
+    (v128.store offset=16 (i32.const 1) (local.get $1))
+    (v8x16.shuffle1 (v128.load (i32.const 0)) (v128.load offset=16 (i32.const 1)))
```

Comment:
Note that the shuffle instructions will probably be renamed relatively soon. See https://github.com/WebAssembly/simd/issues/78.

Reply:
Yes, I knew the potential change, I'd like keep the shuffle here as a placeholder, once it's renamed, I can update it directly.